diff --git a/node_modules/@cosmwasm/ts-codegen/main/utils/schemas.js b/node_modules/@cosmwasm/ts-codegen/main/utils/schemas.js
index 2d75272..f91933e 100644
--- a/node_modules/@cosmwasm/ts-codegen/main/utils/schemas.js
+++ b/node_modules/@cosmwasm/ts-codegen/main/utils/schemas.js
@@ -43,7 +43,7 @@ var readSchemas = /*#__PURE__*/function () {
             fn = clean ? _cleanse.cleanse : function (str) {
               return str;
             };
-            files = (0, _glob.sync)(schemaDir + '/**/*.json');
+            files = (0, _glob.sync)(schemaDir + '/*.json');
             schemas = files.map(function (file) {
               return JSON.parse((0, _fs.readFileSync)(file, 'utf-8'));
             });
